sig-security/sig-security-external-audit/README.md

```diff
@@ -0,0 +1,65 @@
+# SIG Security External Audit Subproject
+
+## Overview
+
+The SIG Security External Audit subproject (subproject, henceforth) is responsible for coordinating regular, comprehensive, third-party security audits.
+The subproject publishes the deliverables of the audit after abiding to the [Security Release Process](https://github.com/kubernetes/security/blob/master/security-release-process.md) and [embargo policy](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy).
+
+  - [Request for Proposal (RFP)](#rfp)
+    - [Security Audit Scope](#security-audit-scope)
+    - [Vendor and Community Questions](#vendor-and-community-questions)
+  - [Review of Proposals](#review-of-proposals)
+  - [Vendor Selection](#vendor-selection)
+  - [Deliverables](#deliverables)
+
+## RFP
+
+The subproject produces a RFP for a third-party, comprehensive security audit. The subproject publishes the RFP in the `kubernetes/community/sig-security` repository. The subproject defines the scope, schedule, methodology, selection criteria, and deliverables in the RFP.
+
+Previous RFPs:
+  - [2019](https://github.com/kubernetes/community/blob/master/sig-security/security-audit-2019/RFP.md)
+  - [2021](https://github.com/kubernetes/community/blob/master/sig-security/security-audit-2021/RFP.md)
+
+As efforts begin for the year's security audit, create a tracking issue for the security audit in `kubernetes/community` with the `sig/security` label.
```

Comment:
Would it make sense to add another label for this so you don't have to sift through all the other sig/security things to find the ones that pertain specifically to the audit?

Reply:
@IanColdwater, good point, how about adding a `/label external-audit`

Reply:
looks like `/label external-audit` is not allowed